web/src/app/shared/models/notifications.ts

```diff
@@ -59,163 +59,194 @@ export enum NOTIFICATION_TYPES {
   WARNING_CANT_COPY
 }
 
-export const notifications = new Map<NOTIFICATION_TYPES, Pair>([
+export const notifications = new Map<NOTIFICATION_TYPES, Pair<string, string>>([
```

Comment:
I see why you would like to have left,right as attr names. Is there any way we can name the attrs meaningfully?